### PR TITLE
Drop OGRE/ from #include directives

### PR DESCRIPTION
**Pull request by rhaschke**
Implements https://github.com/ros-visualization/rviz/pull/1630#issuecomment-1072626521. Untested.

### CODE REVIEW

File: src/rviz/ogre_helpers/movable_text.cpp [hunk 1]
```diff
@@ -41,16 +41,16 @@
...
+#include <OgreVector3.h>
+#include <OgreQuaternion.h>
+#include <OgreRoot.h>
+#include <OgreCamera.h>
+#include <OgreSceneNode.h>
+#include <OgreMaterialManager.h>
+#include <OgreHardwareBufferManager.h>
+#include <Overlay/OgreFontManager.h>
+#include <Overlay/OgreFont.h>
+#include <OgreUTFString.h>
```
Comment (simonschmeisser) [inline src/rviz/ogre_helpers/movable_text.cpp:53]: this makes https://github.com/ros-visualization/rviz/commit/a4919f52fb37da074fcea1648e257691037a3feb superfluous as proposed originally in #1645  :wink:

**Comment by v4hn** [comment]
https://github.com/OGRECave/ogre/pull/2411

File: src/rviz/ogre_helpers/movable_text.cpp [hunk 2]
```diff
@@ -41,16 +41,16 @@
...
-#include <OGRE/Overlay/OgreFont.h>
-#include <OGRE/OgreUTFString.h>
+#include <OgreVector3.h>
+#include <OgreQuaternion.h>
+#include <OgreRoot.h>
+#include <OgreCamera.h>
+#include <OgreSceneNode.h>
+#include <OgreMaterialManager.h>
+#include <OgreHardwareBufferManager.h>
+#include <Overlay/OgreFontManager.h>
```
Comment (paroj) [inline src/rviz/ogre_helpers/movable_text.cpp:51]: CMake will add Overlay to the include path, so you should not use the `Overlay/` prefix.

Also, there is native movable text in 1.12.10+:
https://www.ogre3d.org/2020/12/13/ogre-1-12-10-released#3dtext

**Review by v4hn — APPROVED** [review]
As stressed by upstream in the linked PR, Ogre headers must never be included with a folder prefix.

So this patch makes sense after also removing all the `Overlay/` references.

**Pull request merged**